renovate.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": [
     "config:base",  ":preserveSemverRanges"
-  ]
+  ],
+  "includeForks": true
```

Comment:
@rarkins I'm puzzled that an automated "deps" run would be pushing a repo policy change as well.  Is this intentional?  Is there documentation

Reply:
Somehow I committed that by mistake when troubleshooting yesterday. It's not normal or intentional, however harmless. Can you try the rebase/retry checkbox?

Reply:
I've force pushed to remove my commit from the master branch of https://github.com/renovate-bot/python-bigtable. If you tick the rebase/retry checkbox above then it hopefully will fix up the PR too.

Reply:
I've checked the "retry" box.